README.md

```diff
@@ -2,4 +2,43 @@
 
 Cell Painting and other pipelines from the Imaging Platform.
 
-**Note**: The `without_batchfile` version of a pipeline is identical to the version without the suffix, except that the `CreateBatchFiles` module is disabled. The `without_batchfile` version exists because [Distributed-CellProfiler](https://github.com/CellProfiler/Distributed-CellProfiler) currently does not support the use of batch files. When updating the pipeline, ensure that these two versions are always in sync. 
+## Guidelines for creating pipelines 
+
+1. For each pipeline (e.g. `analysis.cpppipe`), always create a `without_batchfile` version of a pipeline (e.g. `analysis_without_batchfile.cpppipe`) and store in the same directory. The `without_batchfile` version is identical to the version without the suffix, except that the `CreateBatchFiles` module is disabled. The `without_batchfile` version exists because [Distributed-CellProfiler](https://github.com/CellProfiler/Distributed-CellProfiler) does not support the use of batch files. When updating the pipeline, ensure that these two versions are always in sync. 
```

Comment:
Not quite- it's because CellProfiler's behavior when running with CreateBatchfile is completely different than without (in ways I can explain but that I think are beyond the scope of what you want to go into here).

Reply:
Good point – that note was misleading.

I'll update

Reply:
I'll update to

Each pipeline has two versions – one with `CreateBatchFiles` module enabled (named e.g. `analysis.cppipe`), and the other with the moduled disabled (named e.g. `analysis_without_batchfile.cppipe`). When updating the pipeline, ensure that these two versions are always in sync. 
The former is required by a [script](https://github.com/broadinstitute/cellpainting_scripts/blob/master/create_batch_files.sh) for creating CellProfiler batch files, which in turn are used to create a list of groups of images to be processed together, as well as Distributed-CellProfiler configuration files. The latter is used to run the pipeline using [Distributed-CellProfiler](https://github.com/CellProfiler/Distributed-CellProfiler/wiki).